apps/sensenet/src/components/dialogs/change-password.tsx

```diff
@@ -0,0 +1,118 @@
+import { useLogger, useRepository } from '@sensenet/hooks-react'
+import { Button, DialogActions, DialogContent } from '@material-ui/core'
+import TextField from '@material-ui/core/TextField'
+import React, { useState } from 'react'
+import { useCurrentUser } from '../../context'
+import { useGlobalStyles } from '../../globalStyles'
+import { useLocalization } from '../../hooks'
+import { DialogTitle, useDialog } from '.'
+
+export function ChangePasswordDialog() {
+  const { closeLastDialog } = useDialog()
+  const localization = useLocalization().changePassword
+  const logger = useLogger('change-password')
+  const globalClasses = useGlobalStyles()
+  const repo = useRepository()
+  const currentUser = useCurrentUser()
+  const [oldPassword, setOldPassword] = useState<string>()
+  const [newPassword, setNewPassword] = useState<string>()
+  const [confirmPassword, setConfirmPassword] = useState<string>()
+  const [match, setMatch] = useState<boolean>()
```

Comment:
should be true by default, because error is already shown when the form is opened
even better would be if we check for input fields' dirtiness (input was already focused before), and only check when both input is dirty